priv/erlang.config.in

```diff
@@ -38,6 +38,15 @@
    {crash_log, "priv/log/crash.log"}
   ]},
 
+ {ssl, 
+  [
+%%% The number of maximum number of ssl sessions cached by the server. Increase if you 
+%%% of incoming connections.
```

Comment:
You can't run out of them, it is just a cache with session data. When the max is reached it will invalidate all entries.